### ISSUE TITLE
Error: No first tag

### ISSUE THREAD

**Issue by zobak9** [johannesjo/super-productivity#503]
### Steps to Reproduce

1. Click "Add new task" on Today screen. Input box appears
2. Start typing - bug occurs
3.
4.

### Stacktrace
```
t.<anonymous> (webpack:///src/app/features/tasks/add-task-bar/add-task-bar.component.ts:251:14)
Ur (file:///C:/Program%20Files/WindowsApps/53707johannesjo.SuperProductivity_5.5.2.0_x64__ch45amy23cdv6/app/resources/app.asar/dist/main.7239029bd4c141615c72.js:1:3210781)
Ur (file:///C:/Program%20Files/WindowsApps/53707johannesjo.SuperProductivity_5.5.2.0_x64__ch45amy23cdv6/app/resources/app.asar/dist/main.7239029bd4c141615c72.js:1:3210886)
Ur (file:///C:/Program%20Files/WindowsApps/53707johannesjo.SuperProductivity_5.5.2.0_x64__ch45amy23cdv6/app/resources/app.asar/dist/main.7239029bd4c141615c72.js:1:3209821)
ZoneAwarePromise (webpack:///node_modules/zone.js/dist/zone.js:913:32)
Ur (file:///C:/Program%20Files/WindowsApps/53707johannesjo.SuperProductivity_5.5.2.0_x64__ch45amy23cdv6/app/resources/app.asar/dist/main.7239029bd4c141615c72.js:1:3209566)
Ur (file:///C:/Program%20Files/WindowsApps/53707johannesjo.SuperProductivity_5.5.2.0_x64__ch45amy23cdv6/app/resources/app.asar/dist/main.7239029bd4c141615c72.js:1:3216485)
_getCtxForTaskSuggestion (webpack:///src/app/features/tasks/add-task-bar/add-task-bar.component.ts:311:29)
map (webpack:///src/app/features/tasks/add-task-bar/add-task-bar.component.ts:309:27)
```

### Meta Info
META: SP5.5.2 Electron – sk – Win32 – Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) superProductivity/5.5.2 Chrome/83.0.4103.122 Electron/9.1.0 Safari/537.36

**Comment by johannesjo** [comment]
Thanks for reporting. I try to check this tomorrow, but I think that this should be already fixed with the newest release.

**Comment by johannesjo** [comment]
Yes this should be fixed since 5.5.5.
